pokefight/fetch_all_species.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+
+import requests
+
+BASE_URL = "https://pokeapi.co/api/v2/pokemon-species"
+BASE_SPRITE_URL = (
+    "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon"
+)
+
+
+def get_pokemon_sprite_url(*, pokemon_id):
+    return f"{BASE_SPRITE_URL}/{pokemon_id}.png"
+
+
+def fetch_all_species(*, output_file):
+    all_species = []
+    for pokemon_id in range(1, 151):
+        names = requests.get(f"{BASE_URL}/{pokemon_id}").json()["names"]
+
+        all_species.append(
+            {
+                "pokemon_id": pokemon_id,
+                "names": {name["language"]["name"]: name["name"] for name in names},
+                "img_url": get_pokemon_sprite_url(pokemon_id=pokemon_id),
+            }
+        )
+
+    with open(output_file, "w") as f:
+        json.dump(all_species, f)
+
+
+if __name__ == "__main__":
+    output_file = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+        "data/pokemon_species.json",
+    )
```

Comment:
You should try pathlib :)
```suggestion
    output_file = (
    	pathlib.Path(__file__).parents[2] / "data/pokemon_species.json"
    )
```